core/src/main/java/greencity/service/impl/AdviceTranslationServiceImpl.java

```diff
@@ -63,7 +63,10 @@ public AdviceTranslationServiceImpl(AdviceTranslationRepo adviceTranslationRepo,
             }.getType());
         adviceTranslations = adviceTranslations
             .stream()
-            .peek(a -> a.setAdvice(advice))
+            .map(a -> {
+                a.setAdvice(advice);
```

Comment:
Is that really reported by Sonar? You can do it even better instead of reassigning a newly collected `List` into `adviceTranslations` once again and leaving the old one for Garbage Collector:

`adviceTranslations.stream().forEach(a -> a.setAdvice(advice));`

But please check if it works.